ProcessingShowPath/ProcessingShowPath.pde

```diff
@@ -38,33 +79,55 @@ void draw() {
     stop();
     noLoop();
   }
-  
 }
+
+void drawPath()
+{
+  try {
+    while((line = reader.readLine())!= null)
+    {
+      lineDraw(line);
+    }
+    System.out.println(forwardTime + ", " + backTime + ", " + mistakes + ", " + shortcuts);
+    System.out.println("File read");
+  }
+  catch (IOException e)
+  {
+    e.printStackTrace();
+  }
+}
+
+
 void lineDraw(String line){
   String[] coords = split(line, " ");
   if(!coords[0].equals("EndTaskCompleted"))  //If this is not the end task tag
   {
     if(coords.length > 2)    //if coordinates, not mistake and shortcut count
     {
-      int x = Integer.parseInt(split(coords[0], ",")[0]);
-      int y = Integer.parseInt(split(coords[1], ",")[0]);
-      int z = Integer.parseInt(split(coords[2], ",")[0]);
-      
-      pixels[(600 - z)*800 + x] = c;
-      //pixels[(600 - z)*800 + x*2 + 1] = c;
-      //pixels[(600 - z)*800 + x*2 + 800 + 1] = c;
-      //pixels[(600 - z)*800 + x*2 + 800] = c;
-      updatePixels();
-      
-      if(!forwardTimeSaved)
-      {
-         coords[7] = coords[7].replace(',', '.');
-         forwardTime = Float.parseFloat(coords[7]);
-      }
-      if(!backTimeSaved)
+      if(c == forwardColor && drawForward || c == backColor && drawBack)
       {
-         coords[7] = coords[7].replace(',', '.');
-         backTime = Float.parseFloat(coords[7]) - forwardTime;
+        coords[0] = coords[0].replace(',', '.');      //Processing only likes floats with "." and not ","
+        coords[2] = coords[2].replace(',', '.');      //Processing only likes floats with "." and not ","
+        int x = int(startBiasX + (stretchMultiplierX * Float.parseFloat(coords[0])));
+        //int y = Integer.parseInt(split(coords[1], ",")[0]);
```

Comment:
not needed?

Reply:
It is needed, I leave y commented in case we want to do something with elevation in the future